.github/workflows/ci-build.yml

```diff
@@ -20,7 +20,11 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with: 
           version: v1.29
-          working-directory: gomanifest   
+          working-directory: gomanifest
+          #args: --enable-all
+      - name: Linter
```

Comment:
We are adding Separate linter, in other words, two linters will exists parallelly, Why are we nesting one in another..?

Reply:
do you want me to have it as a seprate job/step ?

Reply:
done